extensions/go-package-search/package.json

```
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "go-package-search",
+  "title": "Go Package Search",
+  "description": "Quickly search packages on https://pkg.go.dev",
+  "icon": "command-icon.png",
+  "author": "bstncartwright",
+  "categories": [
+    "Developer Tools",
+    "Productivity",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Go Packages",
+      "subtitle": "pkg.go.dev",
```

```suggestion
      "title": "Search Packages",
      "subtitle": "Go",
```
Could this be a better idea?

---

sounds great, thanks!